Shared/mods/deathmatch/logic/lua/CLuaTimer.cpp

```diff
@@ -68,5 +68,9 @@ CTickCount CLuaTimer::GetTimeLeft()
 {
     CTickCount llCurrentTime = CTickCount::Now();
     CTickCount llTimeLeft = m_llStartTime + m_llDelay - llCurrentTime;
+    
+    if (llTimeLeft.ToLongLong() < 0)
+        return CTickCount(0LL);
+    
     return llTimeLeft;
```

Comment:
What about using the ternary operator?
We could combine the logic into a single return statement.

```suggestion
    return llTimeLeft.ToLongLong() < 0 ? CTickCount(0LL) : llTimeLeft;
```